tests/runtimes/simple-keyvalue/src/main.rs

```diff
@@ -26,19 +34,424 @@ use oasis_core_runtime::{
         state::staking::ImmutableState as StakingImmutableState,
     },
     rak::RAK,
-    register_runtime_txn_methods, runtime_context,
+    runtime_context,
     storage::{StorageContext, MKVS},
     transaction::{
-        dispatcher::{BatchHandler, CheckOnlySuccess},
+        dispatcher::{Dispatcher, ExecuteBatchResult, ExecuteTxResult},
+        tags::Tags,
+        types::{TxnBatch, TxnCall, TxnCheckResult, TxnOutput},
         Context as TxnContext,
     },
-    version_from_cargo, Protocol, RpcDemux, RpcDispatcher, TxnDispatcher, TxnMethDispatcher,
+    types::{CheckTxResult, Error as RuntimeError},
+    version_from_cargo, Protocol, RpcDemux, RpcDispatcher, TxnDispatcher,
 };
 use simple_keymanager::trusted_policy_signers;
 use simple_keyvalue_api::{
     with_api, AddEscrow, Key, KeyValue, ReclaimEscrow, Transfer, UpdateRuntime, Withdraw,
 };
 
+// This is the old runtime method dispatcher that used to be part
+// of the oasis-core/runtime API.  It has been moved here since
+// new runtimes should use the SDK instead.
+//
+// Note: I'm too stupid to figure out how to make this into a separate
+// module, and I don't care enough to put more than "I tried it and
+// everything broke" amounts of effort into it.
```

Comment:
Hm, what was the issue?

Reply:
Importing it by trying to abuse `#[path = "transaction.rs"]` from the upgraded version didnt' work because of the macro or something.